packages/builder/src/internal/core/errors-list.ts

```diff
@@ -452,21 +452,21 @@ Please double check how you invoked algob or run your task.`,
     },
   },
   BUILTIN_TASKS: {
-    RUN_FILE_NOT_FOUND: {
+    RUN_FILES_NOT_FOUND: {
       number: 601,
-      message: "Script %script% doesn't exist.",
-      title: "Script doesn't exist",
-      description: `Tried to use \`algob run\` to execute a non-existing script.
+      message: "Scripts don't exist: %scripts%.",
+      title: "Scripts don't exist.",
+      description: `Tried to use \`algob run\` to execute a non-existing script(s).
```

Comment:
this message doesn't make much sense. A user is already running `algob run`. 
Also, why we are removing a filename from the message?

Reply:
This message appears when user specifies the scripts that he wants to execute:
`algob run 1 2 3`.
When any of those don't exist they are displayed in this message.
If scripts 2 and 3 don't exist they will be displayed this way:
`Scripts don't exist: 1,2`

Reply:
OK. Good.
We shall abort running all scripts, when any of them doesn't exists.